csrc/expr_evaluator.cpp

```diff
@@ -262,7 +263,8 @@ void ExpressionEvaluator::propagateBoundValuesThroughExactMaps(Fusion* fusion) {
   // We map Symbolic IterDomains here only if their extents match. This avoids
   // mapping between symbolic domains that might concretize to an (Iteration,
   // Broadcast) pair from a resolved broadcast.
-  const auto mapped_sets = ExactRootDomainMap(fusion).getMappedSets();
+  const auto mapped_sets =
```

Comment:
Unrelated but to avoid a copy

Reply:
Does it? I suspect the std::move here does nothing because `getMappedSets` returns a **const** reference. 

```suggestion
  ExactRootDomainMap domain_map(fusion);
  const DisjointSets<const IterDomain*>& mapped_sets = domain_map.getMappedSets();
```

Reply:
You're right. I'll take this change off and create a separate PR.